### PR TITLE
Feature project detail view

### LINKED ISSUE #50: Manager Base Interface
- [ ]  Dashboard with available projects
- [ ]  Interface with people assigned to projects (feature moved to creating raport feature)
- [ ]  Interface with all details of chosen project

Manager Base Interface is directed to the Admin-type and Manager-type user
- for Admin-type user:
- [x] **List** with available projects
- [x] Interface with all **details** of chosen project
- [x] Interface for **updating** details of chosen project
- [ ] Interface for **creating** new project
- [ ] Interface for **deleting** project

- for Manager-type user:
- [x] **List** with manager's projects
- [x] Interface with all **details** of manager's project
- [x] Interface for **updating** details of manager's project
- [ ] Interface for **deleting** manager's project

### LINKED ISSUE #51: Create tests for projects views
Create necessary tests for managers app views

- [ ] Tests for ProjectsList APIviews

- [ ] Tests for ProjectsDetail APIviews

- [ ] Tests for ProjectsUpdateAPIviews

- [ ] Tests for ProjectsCreate APIviews

- [ ] Tests for ProjectsDelete APIviews

### PR DESCRIPTION
**Pull request by MartynaAnnaGottschling**
Resolves #50
Resolves #51

### CODE REVIEW

File: managers/templates/managers/project_detail.html [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+      <span class="glyphicon glyphicon-pencil"></span>
+    </a>
+  </small>
+</h2>
+
+<br/>
+
+<u1>
+  <b>Started at: </b> {{project.start_date}}
+  {% if project.terminated is True %}
```
Comment (rwrzesien) [inline managers/templates/managers/project_detail.html]: It is enought to do `{% if project.terminated %}`

File: managers/templates/managers/project_detail.html [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    &nbsp; <font color="Grey"> terminated</font>
```
Comment (rwrzesien) [inline managers/templates/managers/project_detail.html]: Please add translation here and below.

File: managers/tests/test_unit_managers_apiview.py [hunk 3]
```diff
@@ -36,6 +36,7 @@ def setUp(self):
         self.project.members.add(self.user)
         self.url = [
             reverse('custom-projects-list'),
+            reverse('custom-project-detail', args=(self.project.pk,)),
```
Comment (rwrzesien) [inline managers/tests/test_unit_managers_apiview.py]: Please keep it as string, and overwrite in subclass `setUp()` method.

File: managers/views.py [hunk 4]
```diff
@@ -34,3 +35,12 @@ def get(self, request):
         return Response({
             'projects_queryset': projects_queryset,
         })
+
+
+class ProjectDetail(APIView):
+    renderer_classes = [renderers.TemplateHTMLRenderer]
+    template_name = 'managers/project_detail.html'
```
Comment (rwrzesien) [inline managers/views.py]: OK I think this is what we discussed on last meeting about Sheetstorm code review, but we couldn't get common understanding. So you want to use endpoints but actually return generated html code - is that correct? If yes, then are you planning on replacing whole html code with the received response?

File: managers/tests/test_unit_managers_apiview.py (hunk 3 above)
Comment (kbeker) [reply managers/tests/test_unit_managers_apiview.py]: Or change it into separate variables in `setUp()` like I said here: https://github.com/Code-Poets/sheetstorm/pull/85#discussion_r264261234  Choice is yours :)

File: managers/templates/managers/project_detail.html (hunk 1 above)
Comment (MartynaAnnaGottschling) [reply managers/templates/managers/project_detail.html]: Done

File: managers/templates/managers/project_detail.html (hunk 2 above)
Comment (MartynaAnnaGottschling) [reply managers/templates/managers/project_detail.html]: Done

File: managers/tests/test_unit_managers_apiview.py (hunk 3 above)
Comment (MartynaAnnaGottschling) [reply managers/tests/test_unit_managers_apiview.py]: Done

**Pull request merged**